common/lib/dependabot/pull_request_creator/message_builder.rb

```diff
@@ -64,7 +64,7 @@ def library_pr_name
         pr_name +
           if dependencies.count == 1
             "#{dependencies.first.display_name} requirement "\
-            "from #{old_library_requirement(dependencies.first)} "\
+            "#{from_version_msg(old_library_requirement(dependencies.first))}"\
```

Comment:
This is a pretty big change, we could try and figure out the previous version by taking the lowest previous ref if possible to compare (e.g. v1 and v2) otherwise falling back to the first one if there are multiple